frontend/src/Components/AlgInputs/AlgInputsCBA.js

```diff
@@ -107,7 +107,10 @@ class AlgInputsCBA extends Component {
 
     setLoadForecastResults = () => {
         const processedLoadForecastResults = processResults(this.state.loadForecastResults);
-        this.setState({ chartTitles: [`${this.state.profileName}: Uncontrolled`, `${this.state.profileName}: Controlled`] });
+        const profileMatch = this.state.profileData.filter((profile) => profile.config_name === this.state.profileName)[0];
```

Comment:
If the `config/load_forecast` request fails or doesn't return any data, this line will throw an exception. Worth handling it.

Reply:
Added in error handling